tox.ini

```diff
@@ -4,8 +4,18 @@ skip_missing_interpreters = true
 
 
 [testenv]
+passenv =
+    USERNAME
+    LOCALAPPDATA
+setenv =
+    MPLBACKEND=agg
 deps =
     -r{toxinidir}/requirements.txt
```

Comment:
```suggestion
    -r{toxinidir}/requirements.txt
```

Is it possible to remove this?

Reply:
oh... you mean that the dependency in the requirements.txt should be installed in the package installation phase, right? let me check that again. thanks.

Reply:
Yes, for example, `pip install datumaro` doesn't require `pip -r requirements.txt` explicitly.

Reply:
updated.